26-register_publication/creator_navigates_to_the_description_tab_and_handles_projects.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Creator navigates to the description tab and handles Projects
+
+  @445
+  Scenario: Creator searches for Project
+    Given Creator begins registering a Registration in the Wizard
+    When they navigate to the Description tab
+    And they see input field for Projects
+    And they enter search term in Projects field
+    Then they see list of Projects matching the search term
+    And they see associated institutions for each Project
```

Comment:
```suggestion
    And they see a Search box for Projects
    And they enter search term in the Search box
    Then they see list of Projects matching the search term
    And they see associated Institutions for each Project
```

Skal man ikke se navnet på prosjektet også? ikke bare associated institutions?

Reply:
neinei, less is more 🙃